src/test/java/seedu/letsgethired/logic/parser/InternTrackerParserTest.java

```diff
@@ -13,10 +14,7 @@
 
 import org.junit.jupiter.api.Test;
 
-import seedu.letsgethired.logic.commands.AddCommand;
-import seedu.letsgethired.logic.commands.ClearCommand;
-import seedu.letsgethired.logic.commands.DeleteCommand;
-import seedu.letsgethired.logic.commands.EditCommand;
+import seedu.letsgethired.logic.commands.*;
```

Comment:
Import statement issue here too! Saw this in a few other places as well! Perhaps we could try using the IntelliJ Checkstyle Plugin to catch these!